testfiles/10xSuite/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+#
+
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgFullServiceRestaurantNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgHospitalNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgLargeHotelNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgLargeOfficeNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgMediumOfficeNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgMidriseApartmentNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgOutPatientNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgPrimarySchoolNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgQuickServiceRestaurantNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgSecondarySchoolNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgSmallHotelNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgSmallOfficeNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgStand-aloneRetailNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgStripMallNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgSuperMarketNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_RefBldgWarehouseNew2004_Chicago.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 10xSuite/10x_VetBuilding.idf VetBuilding.epw)
```

Comment:
So with any of these non-TMY weather data files, we need to be sure we are able to have them posted.  @XuanLuoLBL can you just try changing all these to Chicago and running them all and making sure things are OK?  It would be great if we could just avoid any issues by using our standard weather data files rather than weather data from other sources.

Reply:
@Myoldmopar I run these four large files with the Chicago epw. Three of them work as modified in the commits, one still has some severe error so currently removed. thanks for checking.